test/controllers/articles_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class ArticlesControllerTest < ActionController::TestCase
+  setup do
+    @article = articles(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:articles)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create article" do
+    assert_difference('Article.count') do
+      post :create, article: { content: @article.content, title: @article.title }
+    end
+
+    assert_redirected_to article_path(assigns(:article))
+  end
+
+  test "should show article" do
+    get :show, id: @article
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get :edit, id: @article
+    assert_response :success
+  end
+
+  test "should update article" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.